CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via [issues](https://github.com/KuriusMTL/HackItForward/issues).
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Fork the repository
+2. Clone your fork
+3. Create a branch by running `git checkout -b <branch-name>`
+4. Contribute!
+5. If you get things working, add your changed files with `git add` and run `git commit -m "insert message here"` to commit your messages
+6. Push your changes to your fork with `git push origin <branch-name>`
+7. Create a pull request
+8. Iterate on the solution
+9. Get merged! 🎉 🎊
+
+## Styleguides
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+
+### Python Style Guide
+
+All Python code is formatted with [Black](https://github.com/psf/black).
+
+* Cut off lines at 100 characters
+* Indent with 2 spaces
+
+
+### HTML Style Guide
+
+* Indent with 2 spaces
+
+
+## Code of Conduct
```

Comment:
I'm not a huge fan of the Contributor Covenant, especially the multiple specific clauses followed by a more generic catch-all. I'll see if I can write something we all agree on that's shorter and better.

Reply:
Ok, but for now it will suffice

Reply:
Is there some rush to get this merged? Even if there is, the default should be no code of conduct, not something we'll change within the week.